app/src/main/java/org/simple/clinic/main/TheActivity.kt

```diff
@@ -338,4 +347,13 @@ class TheActivity : AppCompatActivity(), TheActivityUi {
         .setPositiveButton(R.string.deeplink_no_patient_positive_action, null)
         .show()
   }
+
+
+  private fun showTeleconsultNotAllowedErrorDialog() {
+    AlertDialog.Builder(this, R.style.Clinic_V2_DialogStyle)
+        .setTitle("You are not authorised to log teleconsult")
+        .setMessage("Please check with your supervisor")
+        .setPositiveButton("Okay", null)
```

Comment:
Can we not use hardcoded strings in the source code? These should be added as string resources.